androidartifact/apk_info_test.go

```diff
@@ -14,73 +14,82 @@ import (
 func TestParsePackageInfos(t *testing.T) {
 
 	tests := []struct {
-		name    string
-		aaptOut string
-		want    string
-		want1   string
-		want2   string
+		name        string
+		aaptOut     string
+		isAAB       bool
+		packageName string
+		versionCode string
+		versionName string
 	}{
 		{
-			name:    "test_with_empty_platformBuildVersionName",
-			aaptOut: `package: name='com.example.birmachera.myapplication' versionCode='1' versionName='1.0' platformBuildVersionName=''`,
-			want:    "com.example.birmachera.myapplication",
-			want1:   "1",
-			want2:   "1.0",
+			name:        "test_with_empty_platformBuildVersionName",
+			aaptOut:     `package: name='com.example.birmachera.myapplication' versionCode='1' versionName='1.0' platformBuildVersionName=''`,
+			packageName: "com.example.birmachera.myapplication",
+			versionCode: "1",
+			versionName: "1.0",
+		},
+		{
+			name:        "test_without_platformBuildVersionName",
+			aaptOut:     `package: name='com.example.birmachera.myapplication' versionCode='1' versionName='1.0'`,
+			packageName: "com.example.birmachera.myapplication",
+			versionCode: "1",
+			versionName: "1.0",
 		},
 		{
-			name:    "test_without_platformBuildVersionName",
-			aaptOut: `package: name='com.example.birmachera.myapplication' versionCode='1' versionName='1.0'`,
-			want:    "com.example.birmachera.myapplication",
-			want1:   "1",
-			want2:   "1.0",
+			name:        "test_with_platformBuildVersionName",
+			aaptOut:     `package: name='com.example.birmachera.myapplication' versionCode='1' versionName='1.0' platformBuildVersionName='3'`,
```

Comment:
Which version of the bundletool did you test with? The output of that tool may have changed. We set the version here: https://github.com/bitrise-steplib/steps-deploy-to-bitrise-io/blob/bbc213a9d8c1ab161c7b31a6fa7660e847e2849c/step.yml#L187.

Reply:
Initially I tried it with 1.11.0 and the latest version `1.14.1`. But after your comment I also checked `1.8.1` and it uses the same `package` key for the package name. 

One thing I would mention is that the preinstalled version:
https://github.com/bitrise-io/bitrise.io/blob/e27882b1a506684f960120b044277a5984fcb82f/system_reports/LINUX/linux-docker-android-20.04.log#L283-L284

And the `1.8.1` default tool version are both pretty old ones. Maybe it is time to upgrade them.

Reply:
Yes, we can just use the preinstalled version to avoid downloading bundletool every time.